genrl/agents/deep/a2c/a2c.py

```diff
@@ -193,17 +194,16 @@ def get_hyperparams(self) -> Dict[str, Any]:
             "lr_policy": self.lr_policy,
             "lr_value": self.lr_value,
             "rollout_size": self.rollout_size,
-            "weights": self.ac.state_dict(),
         }
-        return hyperparams
+        return hyperparams, self.ac.state_dict()
 
-    def load_weights(self, weights) -> None:
+    def _load_weights(self, weights) -> None:
```

Comment:
Any reason to add the `_`?

Reply:
it has the same name as the parameter and we are not going to let user get to this :P